ios/RNZoomUs.m

```diff
@@ -215,6 +215,200 @@ - (void)connectAudio {
+  }
+}
+
+RCT_EXPORT_METHOD(muteMyVideo: (BOOL)muted resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+  @try {
+    MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+    if (!ms) {
+      reject(@"ERR_ZOOM_MEETING_CONTROL", @"Meeting has not start", nil);
+      return;
+    }
+    MobileRTCVideoError error = [ms muteMyVideo:muted];
+    if (error == 0) {
+      resolve(nil);
+    } else {
+      reject(@"ERR_ZOOM_MEETING_CONTROL", [NSString stringWithFormat:@"Mute my video error, status: %lu", error], nil);
+    }
+  } @catch (NSError *ex) {
+    reject(@"ERR_UNEXPECTED_EXCEPTION", @"Executing muteMyVideo", ex);
+  }
+}
+
+RCT_EXPORT_METHOD(muteAttendee: (NSString *)userId muted:(BOOL)muted resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+  @try {
+    MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+    if (!ms) {
+      reject(@"ERR_ZOOM_MEETING_CONTROL", @"Meeting has not start", nil);
+      return;
+    }
+    NSNumber *zoomUserId = @([userId intValue]);
+    if ([ms muteUserAudio:muted withUID:zoomUserId]) {
+      resolve(nil);
+    } else {
+      reject(@"ERR_ZOOM_MEETING_CONTROL", @"Fail to mute attendee", nil);
+    }
+  } @catch (NSError *ex) {
+    reject(@"ERR_UNEXPECTED_EXCEPTION", @"Executing muteAttendee", ex);
+  }
+}
+
+RCT_EXPORT_METHOD(muteAllAttendee: (BOOL)allowUnmuteSelf resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+  @try {
+    MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+    if (!ms) {
+      reject(@"ERR_ZOOM_MEETING_CONTROL", @"Meeting has not start", nil);
+      return;
+    }
+    if ([ms muteAllUserAudio: allowUnmuteSelf]) {
+      resolve(nil);
+    } else {
+      reject(@"ERR_ZOOM_MEETING_CONTROL", @"Fail to mute all attendee", nil);
+    }
+  } @catch (NSError *ex) {
+    reject(@"ERR_UNEXPECTED_EXCEPTION", @"Executing muteAllAttendee", ex);
+  }
+}
+
+RCT_EXPORT_METHOD(startShareScreen: (RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+  @try {
+    MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+    if (!ms) {
+      reject(@"ERR_ZOOM_MEETING_CONTROL", @"Meeting has not start", nil);
+      return;
+    }
+    if ([ms startAppShare]) {
+      resolve(nil);
+    } else {
+      reject(@"ERR_ZOOM_MEETING_CONTROL", @"Fail to share screen", nil);
+    }
+  } @catch (NSError *ex) {
+    reject(@"ERR_UNEXPECTED_EXCEPTION", @"Executing startShareScreen", ex);
+  }
+}
+
+RCT_EXPORT_METHOD(stopShareScreen: (BOOL)muted resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+  @try {
+    MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+    if (ms) {
+      [ms stopAppShare];
+    }
+    resolve(nil);
+  } @catch (NSError *ex) {
+    reject(@"ERR_UNEXPECTED_EXCEPTION", @"Executing stopShareScreen", ex);
```

Comment:
I've noticed that some rejections are not consistent between ios and android
android: `promise.reject("ERR_ZOOM_MEETING_CONTROL", "Stop share screen error, status: " + result.name());`